style.css

```diff
@@ -0,0 +1,253 @@
+  max-width: 40px;
+  max-height: 40px;
+  margin: 5px;
+  padding: 5px;
+}
+svg:hover {
+  border-radius: 50%;
+  background-color: rgba(29, 156, 240, 0.266);
+}
+ion-icon {
+  margin-right: 10px;
+  max-height: fit-content;
+  position: relative;
+}
+.left-menu {
+  padding: 10px;
+  font-size: 20px;
+}
+.left-menu:hover {
+  border-radius: 20px;
+  background-color: rgba(128, 128, 128, 0.445);
+}
+.tweet {
+  border: 1px solid rgb(29, 155, 240);
+  background-color: rgb(29, 155, 240);
+  color: white;
+  border-radius: 20px;
+  font-size: 15px;
+  text-align: center;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-right: 10px;
+  max-width: 200px;
+}
+.tweet:hover {
+  background-color: rgb(29, 120, 240);
+}
+.header {
+  padding: 5px;
+  border-bottom: 1px solid rgba(128, 128, 128, 0.637);
+}
+.left-panel .profile {
+  margin-top: 400px;
+  padding: 10px;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  place-items: center;
+  max-width: 250px;
+  margin-bottom: 20px;
+}
+.profile img {
+  padding-left: 5px;
+  border-radius: 50%;
+}
+.account-name {
+  padding-left: 10px;
+}
+.profile p {
+  color: grey;
+}
+.profile ion-icon {
+  padding-left: 10px;
+}
+.left-panel .profile:hover {
+  border-radius: 40px;
+
+  box-shadow: grey 1px 1px;
+  filter: brightness(90%);
+}
+
+.header {
+  position: relative;
+}
+.header ion-icon {
+  position: absolute;
+  top: 30%;
+  right: 0;
+}
+.main-content {
+  display: grid;
+  grid-template-rows: auto auto;
+}
+.maketweet-section {
+  position: relative;
+  border-bottom: 1px solid grey;
+}
+.profile-pic img {
+  display: inline-block;
+  border-radius: 50%;
+  margin: 20px;
+  width: 15%;
+  height: 25%;
+}
+.writetweet {
+  display: grid;
+  grid-template-rows: max-content;
+  position: absolute;
+  top: 0;
+  left: 20%;
+  padding-top: 5%;
+  padding-left: 10px;
+}
+.tweet-input {
+  border: none;
+  font-size: xx-large;
+}
+.tweet-input:focus {
+  outline: none;
+}
+.writetweet ion-icon {
+  color: rgb(29, 120, 240);
+  padding: 10px;
+  width: 30px;
+  height: 40px;
+  margin-top: 20px;
+}
+.writetweet ion-icon:hover {
+  background-color: rgba(29, 120, 240, 0.486);
+  border-radius: 50%;
+}
+.writetweet .tweet {
+  margin-left: 80%;
+}
+.tweetfeed {
+  position: relative;
+  border: 1px solid rgba(128, 128, 128, 0.637);
+  grid-template-columns: max-content auto;
+}
+.tweetfeed {
+  display: grid;
+  grid-template-rows: auto;
+}
+
+.right-panel {
+  border-left: 1px solid rgba(128, 128, 128, 0.637);
+}
+.search {
+  padding: 8px 5px;
+  background-color: white;
+  position: relative;
+}
+.search ion-icon {
+  position: absolute;
+  top: 20px;
+  left: 18px;
+}
+.search-bar {
+  min-width: 300px;
+  height: 40px;
+  background-color: rgba(128, 128, 128, 0.332);
+  border: 1px solid rgba(128, 128, 128, 0.637);
+  border-radius: 50px;
+  padding: 8px 16px;
+  padding-left: 48px;
+}
+.search-bar:focus {
+  background-color: white;
+}
+.whats-happening {
```

Comment:
layout here is very good, could maybe add more spacing but the grid is similar to the real thing